popup.html

```diff
@@ -131,32 +131,7 @@
                     </div>
                 </form>
             </div>
-            <div class="transactions-container">
-                <div id="tx0" class="transaction">
-                    <div id="tx-type-0" class="tx-type"></div>
-                    <div id="tx-info-0" class="tx-info"></div>
-                </div>
-                <div id="tx1" class="transaction">
-                    <span id="tx-type-1" class="tx-type"></span>
-                    <span id="tx-info-1" class="tx-info"></span>
-                </div>
-                <div id="tx2" class="transaction">
-                    <span id="tx-type-2" class="tx-type"></span>
-                    <span id="tx-info-2" class="tx-info"></span>
-                </div>
-                <div id="tx3" class="transaction">
-                    <span id="tx-type-3" class="tx-type"></span>
-                    <span id="tx-info-3" class="tx-info"></span>
-                </div>
-                <div id="tx4" class="transaction">
-                    <span id="tx-type-4" class="tx-type"></span>
-                    <span id="tx-info-4" class="tx-info"></span>
-                </div>
-                <div id="tx5" class="transaction">
-                    <span id="tx-type-5" class="tx-type"></span>
-                    <span id="tx-info-5" class="tx-info"></span>
-                </div>
-            </div>
+            <div id = "transactions-container" class="transactions-container" style="overflow:scroll; height:259px;"> </div>

```

Comment:
remove the class and move the styling of the element into the CSS file under #transactions-container

Reply:
Why remove the class though and not move the style under . transactions-container?

Reply:
well, we won't really reuse this class as there's going to be only one container for all transactions and we already use the id to get the element